tests/unit-tests/python/foglamp_test/services/core/api/test_statistics_history.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+import asyncpg
+import asyncio
+from datetime import datetime, timedelta
```

Comment:
unused timedelta

Reply:
its not unused; https://github.com/foglamp/FogLAMP/pull/343/files#diff-96e9d24e7d9d1665b7a7ab8153e8ef87R45 (in actual code you are using storage layer which gives you string hence you do split with `:` and then convert to seconds; Here its timedelta itself